topics/sequence-analysis/tutorials/sars-with-galaxy-on-anvil/tutorial.md

```diff
@@ -0,0 +1,412 @@
+> >
+> {: .solution}
+{: .question}
+
+## Quality Scoring
+
+FastQC is a tool which aims to provide simple quality control checks on raw sequence data coming from high throughput sequencing pipelines. It provides a set of analyses which you can use to get a quick impression of whether your data has any problems of which you should be aware before doing any further analysis. 
+
+> <hands-on-title>Determine the Quality of the Samples</hands-on-title>
+>
+> 1. Find {% tool [FastQC](https://toolshed.g2.bx.psu.edu/view/devteam/fastqc/5ec9f6bceaee) %} in the GENOMIC FILE MANIPULATION: FASTQ Quality Control tool folder.
+>    - {% icon param-file %} *"Raw read data from your current history"*: `VA_sample_forward_reads.fastq`
+> 2. Run the tool.
+>
+{: .hands_on}
+
+The main dash will highlight in green if everything is okay. In the history, you will see the new files turn yellow, then green. If the job fails it will show an error. 
+
+Click on the eye icon {% icon galaxy-eye %} in the new file in the history “FASTQC on data2 Webpage”. 
+
+You will open up a summary report for the sequencing file:
+
+![Screenshot of the FastQC results. The Basic Statistics and Per Base Sequence Quality sections for the report on VA_sample_forward_reads.fastq are visible.](../../images/sars-with-galaxy-in-anvil/13-fastqc-report.png)
+
+> <question-title></question-title>
+>
+> 1. Explore “Basic Statistics”. How many total reads are there? Have any been flagged as poor quality? What is the sequence length?
+> 2. Explore “Per base sequence quality”. Based on the Basic Statistics, is 28-40 a good or bad quality score? 
+> 3. Is it okay to proceed based on the per base sequence quality?
+>
+> > <solution-title></solution-title>
+> >
+> > 1. 43,522 reads. Zero flagged as poor quality. Sequence length is 3-301 base pairs.
+> > 2. 28-40 is pretty good. 
+> > 3. Yes, because the per base sequence quality is good (“in the green”).
+> >
+> {: .solution}
+{: .question}
+
+> <details-title>Learn more about quality scores</details-title>
+>
+> You may be wondering how the fourth line of the .fastq files relates to the quality score above. To save space, the sequencer records an [ASCII character](http://drive5.com/usearch/manual/quality_score.html) to represent scores 0-42. For example 10 corresponds to “+” and 40 corresponds to “I”. FastQC knows how to translate this. This is often called “Phred” scoring.
+> What does 0-42 represent? These numbers, when plugged into a formula, tell us the probability of an error for that base. This is the formula, where Q is our quality score (0-42) and P is the probability of an error:
+>
+>Q = -10 log10(P)
+>
+>Using this formula, we can calculate that a quality score of 40 means only 0.00010 probability of an error!
+>
+> Learn more from the [Quality Control Tutorial FAQs](https://training.galaxyproject.org/training-material/topics/sequence-analysis/tutorials/quality-control/faqs/).
```

Comment:
This should have triggered an alert but somehow it avoided it. Need to investigate that at some later date.